pkg/apis/messaging/v1alpha1/sequence_validation_test.go

```diff
@@ -144,27 +147,28 @@ func TestSequenceSpecValidation(t *testing.T) {
 		ts: &SequenceSpec{
 			ChannelTemplate: validChannelTemplate,
 			Steps:           []SubscriberSpec{{URI: &subscriberURI}},
-			Reply: &corev1.ObjectReference{
-				APIVersion: "messaging.knative.dev/v1alpha1",
-				Kind:       "inmemorychannel",
+			Reply: &v1alpha1.Destination{
```

Comment:
I understood that we were not going to change the Reply yet because it hadn't really been discussed yet? For subscription I'm planning on leaving the Reply AS-IS for now and then we can discuss that within the context of the v1beta1 task force.

Reply:
NM, this is right.